nng/src/core/taskq.c

```diff
@@ -232,11 +235,29 @@ nni_task_fini(nni_task *task)
 	nni_mtx_fini(&task->task_mtx);
 }
 
+void
+nni_taskq_setter(int num_taskq_threads, int max_taskq_threads) {
+	if (num_taskq_threads)
```

Comment:
Here we need condition which is more strict. for example 16~128. fix it in next commit.